src/MSBuild/Microsoft.Build.xsd

```diff
@@ -5,7 +5,7 @@ xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
 elementFormDefault="qualified">
 
     <!-- =================== IMPORT COMMON SCHEMA =========================== -->
-    <xs:include schemaLocation="MSBuild\Microsoft.Build.CommonTypes.xsd"/>
+    <xs:include schemaLocation="Microsoft.Build.CommonTypes.xsd"/>
```

Comment:
In the [internal source that controls the final VS layout](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=%2Fsrc%2FSetupPackages%2FCoreIde%2FCommunity%2Fres%2Ffiles.swr&version=GBmaster&line=203&lineEnd=211&lineStartColumn=1&lineEndColumn=1&lineStyle=plain), CommonTypes is in a subfolder relative to `Microsoft.Build.xsd`:

```
folder "InstallDir:\Xml\Schemas\$(LCID)"
  file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\Xml\Schemas\DotNetConfig.xsd"
  file source="!(bindpath.sources)\env\msenv\Schemas\snippetformat.xsd"
  file source="!(bindpath.sources)\env\msenv\Schemas\vstemplate.xsd"
  file source="!(bindpath.binaries.x86)\bin\i386\xmake\Microsoft.Build.xsd"
  folder "MSBuild"
    file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\xmake\MSBuild\Microsoft.Build.Commontypes.xsd"
    file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\xmake\MSBuild\Microsoft.Build.Core.xsd"
```

So I think this should be reverted and instead the other files should be placed in a subfolder in the MSBuild.csproj output.